code/game/objects/items/reagent_containers/blood_pack.dm

```diff
@@ -52,3 +52,10 @@
 	name = "Empty BloodPack"
 	desc = "Seems pretty useless... Maybe if there were a way to fill it?"
 	icon_state = "empty"
+
+/obj/item/reagent_containers/blood/OMinus/medic
+	name = "O- emergency bloodbag"
+	desc = "An emergency blood bag meant to be used by syringes to inject blood into patients that have bloodloss. Requires a regular syringe to function."
+	icon_state = "bottle"
```

Comment:
I like the idea, but I would prefer a new sprite for this blood bag, perhaps the current blood bag but thinner.